Models/NHES/Systems/Examples/SMR_highfidelity_4_loop_control_Pressurizer_CR.bak-mo

```diff
@@ -0,0 +1,94 @@
+within NHES.Systems.Examples;
```

Comment:
This is a .bak-mo file that is unpaired with a .mo file. Do we need to change its extension?

Reply:
Taken out.